### PR TITLE
Alonso has age restrictions

### LINKED ISSUE #69108: Underage sex with Alonso Lautrec is possible (and shouldn't be)
### Describe the bug

After looking through his code and verifying if it's possible in a recent playthrough, I've found that there wasn't any way for Alonso to refuse his services to a player's character who is aged 16-17.

### Attach save file

N/A

### Steps to reproduce

1. Create a character with an age variable of 16-17.
2. Spawn or travel to Refugee Center
3. Talk to Alonso and receive his quest
4. Finish quest and talk to him again (with condom in inventory)

### Expected behavior

For Alonso's services to be gated by age.

### Screenshots

_No response_

### Versions and configuration

- OS: Android
    - OS Version: Manufacturer: samsung; Model: SM-S901U; Release: 13; Incremental: S901USQS3CWI1; 
- Game Version: 0997bfc [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth],
    Xedra Evolved [xedra_evolved],
    Bombastic Perks [bombastic_perks],
    Translate Complex Dialogue [translate_dialogue],
    Stats Through Kills [stats_through_kills],
    Stats Through Skills [StatsThroughSkills]
]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
None
#### Purpose of change
Fix #69108
#### Describe the solution
Add math check for player's age
#### Describe alternatives you've considered
Current solution simply cut the dialogue that lead to action; I am open and would be glad to make an additional dialogue that point the reason you can't perform an action, but i am bad at writing, and i don't know Alonso's personality

### CODE REVIEW

**Comment by Maleclypse** [comment]
Thanks for doing this!

File: data/json/npcs/refugee_center/surface_refugees/NPC_Alonso_Lautrec.json [hunk 1]
```diff
@@ -409,6 +429,19 @@
...
+  {
+    "type": "talk_topic",
+    "id": "TALK_REFUGEE_Alonso_Too_Young",
+    "dynamic_line": {
+      "npc_has_var": "Alonso_accent",
+      "type": "general",
+      "context": "none",
+      "value": "asked",
+      "no": "Alonso think maybe you're too young for Alonso.",
+      "yes": "Sorry kid, i can't do this, i still have some rules to follow."
```
Comment (TheShadowFerret) [inline data/json/npcs/refugee_center/surface_refugees/NPC_Alonso_Lautrec.json:441]:
```suggestion
      "yes": "Sorry kid, I can't do this, I still have some rules to follow."
```
This doesn't read to me as "Alonso doesn't want to commit statutory rape," but as "the Free Merchants/Old Guard will throw me in the woodchipper if I did."

File: data/json/npcs/refugee_center/surface_refugees/NPC_Alonso_Lautrec.json [hunk 2]
```diff
@@ -409,6 +429,19 @@
...
+  {
+    "type": "talk_topic",
+    "id": "TALK_REFUGEE_Alonso_Too_Young",
+    "dynamic_line": {
+      "npc_has_var": "Alonso_accent",
+      "type": "general",
+      "context": "none",
+      "value": "asked",
+      "no": "Alonso think maybe you're too young for Alonso.",
+      "yes": "Sorry kid, I can't do this, I still have some rules to follow."
```
Comment (Maleclypse) [inline data/json/npcs/refugee_center/surface_refugees/NPC_Alonso_Lautrec.json:441]:
```suggestion
      "yes": "Sorry kid, I won't do this, you seem too young."
```

File: data/json/npcs/refugee_center/surface_refugees/NPC_Alonso_Lautrec.json (hunk 1 above)
Comment (Maleclypse) [reply data/json/npcs/refugee_center/surface_refugees/NPC_Alonso_Lautrec.json]: You made a good point so I made it more clear Alonso has some ethical hiccups with the idea.

File: data/json/npcs/refugee_center/surface_refugees/NPC_Alonso_Lautrec.json [hunk 3]
```diff
@@ -409,6 +429,19 @@
...
   },
+  {
+    "type": "talk_topic",
+    "id": "TALK_REFUGEE_Alonso_Too_Young",
+    "dynamic_line": {
+      "npc_has_var": "Alonso_accent",
+      "type": "general",
+      "context": "none",
+      "value": "asked",
+      "no": "Alonso think maybe you're too young for Alonso.",
```
Comment (I-am-Erk) [inline data/json/npcs/refugee_center/surface_refugees/NPC_Alonso_Lautrec.json:440]:
```suggestion
      "no": "Alonso thinks maybe we should wait a few years?",
```

File: data/json/npcs/refugee_center/surface_refugees/NPC_Alonso_Lautrec.json [hunk 4]
```diff
@@ -409,6 +429,19 @@
...
+      "yes": "Sorry kid, I won't do this, you seem too young."
```
Comment (I-am-Erk) [inline data/json/npcs/refugee_center/surface_refugees/NPC_Alonso_Lautrec.json:441]:
```suggestion
      "yes": "Sorry kid, you're younger than my little sister was.  Seems wrong."
```

**Review by I-am-Erk — COMMENTED** [review]
just dialogue tweaks

**Pull request merged**